README.md

```diff
@@ -0,0 +1,34 @@
+# Legcord Flatpak
```

Comment:
just to set expectations: no end user will see this.

Reply:
i'm aware